files/es/web/api/batterymanager/dischargingtime/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: BatteryManager.dischargingTime
+slug: Web/API/BatteryManager/dischargingTime
+tags:
+  - API
+  - Apps
+  - Batería
+  - Battery
+  - Firefox OS
+  - Gecko DOM Reference
+  - WebAPI
+  - aplicaciones
+translation_of: Web/API/BatteryManager/dischargingTime
+browser-compat: api.BatteryManager.dischargingTime
+---
+{{deprecated_header}}{{APIRef("Battery API")}}
+
+Indíca la cantidad de tiempo, en segundos,
+que restan antes de que la batería se descargue completamente.
+
+> **Nota:** Incluso si el tiempo devuelto es exacto al segundo, los navegadores los redondean a un intervalo más alto (típicamente a los 15 minutos más cercanos) por razones de privacidad.
+
+## Sintaxis
+
+```js
+var time = battery.dischargingTime;
+```
+
+Valor de retorno, `time` es el tiempo restante en segundos antes que la `battery`,
+que es un objeto {{domxref("BatteryManager")}},
+esté completamente descargada y el sistema se suspenda.
+Este valor es [`Infinito`](/es/docs/Web/JavaScript/Reference/Global_Objects/Infinity)
```

Comment:
Hi, guys.

So far so good, I'd just double check about the `Infinity Object`. Lets say that I code it in Spanish `INFINITO`, will the snippet work property? I think it will trough an unexpected behavior. What you think about letting the word in it's original language? 
```suggestion
Este valor es [`Infinity`](/es/docs/Web/JavaScript/Reference/Global_Objects/Infinity)
```

Reply:
you are right! thank you!